<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/트레이 아이콘.webp</normaloff>icon/트레이 아이콘.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 177, 208);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 30pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>계정 정보 저장</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY중고딕</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;HY중고딕&quot;;</string>
   </property>
   <property name="text">
    <string>아래 입력창에 저장될 이름, 웹사이트 주소, 관련 태그, ID, Password, 사이트에 관한 메모를 기입해주세요.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SaveName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>저장될 이름</string>
   </property>
  </widget>
  <widget class="Line" name="FirstLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>991</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>사이트 주소</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SiteLink">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>120</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>사용자 ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>사용자 PassWord</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>태그</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Tag">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>메모</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Memo">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="FavoriteCheck">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;HY헤드라인M&quot;;</string>
   </property>
   <property name="text">
    <string>즐겨찾기에 추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY중고딕</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;HY중고딕&quot;;</string>
   </property>
   <property name="text">
    <string>예시 : #SNS</string>
   </property>
  </widget>
  <widget class="QLabel" name="warnLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>791</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY중고딕</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;HY중고딕&quot;;</string>
   </property>
   <property name="text">
    <string>이름, 주소, ID, 비밀번호에는 최소 한 글자 이상 입력해주세요.
   중복된 이름은 지양해주세요. 오류가 발생할 수 있습니다</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY중고딕</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;HY중고딕&quot;;</string>
   </property>
   <property name="text">
    <string>태그 입력시엔 #으로 시작해주세요</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
